tests/components/azure_data_explorer/test_init.py

```diff
@@ -117,7 +117,7 @@ async def test_import(hass: HomeAssistant) -> None:
     assert await async_setup_component(hass, DOMAIN, config)
     await hass.async_block_till_done()
 
-    assert "filter" in hass.data[DOMAIN]
+    assert hass.data[DOMAIN]
```

Comment:
Ideally we don't touch the internals but rather see what kind of effect it had on the integration

Reply:
Yes - but I didn't want to touch the tests further

Reply:
But then this test is testing nothing, as it used to test that `filter` is still present in the config

Reply:
You're right - I've removed the test altogether...
The internals are tested inside `test_filter`